src/components/SingleReview.jsx

```diff
@@ -0,0 +1,33 @@
+import { useParams } from "react-router-dom";
+import { getReviewsById } from "../requests";
+import { useEffect } from "react";
+
+const SingleReview = ({ review, setReview }) => {
+  let review_id = useParams().review_id;
+  useEffect(() => {
+    getReviewsById(review_id).then((data) => {
+      setReview(data);
+    });
+  }, []);
+
+  return (
```

Comment:
Just missing a loading state and user feedback for when the review is still being fetched